Source/Modules/php5.cxx

```diff
@@ -528,7 +530,14 @@ class PHP5 : public Language {
     Printf(s_entry, "/* Every non-class user visible function must have an entry here */\n");
     Printf(s_entry, "static zend_function_entry %s_functions[] = {\n", module);
 
+    /* Emit all of the code */
+    Language::top(n);
+
+    SwigPHP_emit_resource_registrations();
+
     /* start the init section */
+    String * s_init_old = s_init;
+    s_init = NewString("/* init section */\n");
```

Comment:
These refactoring changes are not obviously related to the change you say you're making, so without any explanation it looks like you've accidentally included some unrelated change in the commit.  I can guess roughly why you might need this, but only because this is essentially the same change I had to make while adding PHP7 support - someone unfamiliar with that work wouldn't know that.

Probably the best way to address this is to perform this refactor first in a separate commit, then add the new feature in a second commit (the PR would include both commits).